import_map.json

```
@@ -2,6 +2,7 @@
   "imports": {
     "mqtt": "npm:mqtt@4.3.7",
     "mariadb": "npm:mariadb@3.1.1",
-    "oak/": "https://deno.land/x/oak@v12.4.0/"
+    "oak/": "https://deno.land/x/oak@v12.4.0/",
+    "assert/mod.ts": "https://deno.land/std@0.201.0/assert/mod.ts"
```

Ich würd hier ehr die Library inkludieren..

```suggestion
    "std/": "https://deno.land/std@0.201.0/"
```

Und dann in den stellen, wo du das brauchst:

```ts
import { ... } from "std/assert/mod.ts"
```